integration-tests/src/test/resources/sitconfig/configoverrides/diagnostics-WLDF-MODULE-0.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<wldf:wldf-resource
```

Comment:
missing copyright notice

Reply:
Added.

Reply:
Because of the errors I had to remove the copyright notice in config override files.